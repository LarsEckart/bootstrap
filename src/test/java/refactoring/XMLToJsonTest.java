package refactoring;

import java.net.URL;
import org.approvaltests.Approvals;
import org.approvaltests.reporters.UseReporter;
import org.approvaltests.reporters.macosx.BeyondCompareMacReporter;
import org.junit.jupiter.api.Test;

@UseReporter(BeyondCompareMacReporter.class)
class XMLToJsonTest {

  @Test
  void approval() throws Exception {
    XMLToJson xmlToJson = new XMLToJson();
    URL url = getClass().getClassLoader().getResource("xmlToJson.xml");

    String json = xmlToJson.getJson(url, "fk:AMM24_fk:AMM24-FM");

    Approvals.verify(json);
  }
}
