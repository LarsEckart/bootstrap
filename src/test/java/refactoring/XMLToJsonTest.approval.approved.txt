[{'data':'Table of Contents','attr':{'id':'fk:AMM24_fk:AMM24-FM_dk:CTOC-24','file':'CTOC-24.pdf'}}]